README.md

```diff
@@ -1,3 +1,45 @@
 # Memento: The Video Memory Game
 
-This is the repository for Memento, a game that tests your memory on a set of everyday videos. The game is accessible at memento-game.csail.mit.edu .
\ No newline at end of file
+This is the repository for Memento, a game that tests your memory on a set of everyday videos. The game is accessible at [memento-game.csail.mit.edu](https://memento-game.csail.mit.edu).
+
+## Dependencies
+This is a JavaScript project, so you need to have `node` and `npm` installed, and the first time you clone the repo, you need to run `npm i` to install dependencies.
+
+You need to get the `clean_10k.json` dataset file and put it in `task_data/video_datasets`. If you do not have this file and want to test the app, you can edit `api/config.js` and change `clean_10k` to `videomem_300`.
+
+## Mandatory Environment Variables
+The server expects the following environment variables to be set (see `api/config.js`).
+- `MYSQL_HOST`
+- `MYSQL_USER`
+- `MYSQL_PASS`
+- `MEMENTO_ENV` to `test` `dev` or `prod`
+    - set to `test` automatically when running the tests
+    - the database name is ``memento_${MEMENTO_ENV}``
+    - used to pick a config from `api/config.js`
+
+## Commands
+- `npm run dev` run the app; webpack compiles the code every time you save a file
+- `npm run prod` run the app; webpack compiles the code once at the start
```

Comment:
So to be clear, this means we will have to restart the server every time we pull changes to the app on prod? (what we're doing anyway)

Reply:
Yes, what we are doing anyways. Although you might be able to create a command that does a pull, rebuild, and restart.

Reply:
Technically if the change was frontend only you could just pull and rebuild.